server/docker-compose.yml

```diff
@@ -12,6 +12,8 @@ api:
     - INITIAL_ADMIN_CODE=initialadmincode
   links:
    - mongodb
+  labels:
+   - "io.kontena.container.skip_logs=1"
```

Comment:
Yeah, I recommend a separate PR to change these. It's a problem configuring this by default in development without having it configured in production.